### PR TITLE
chunk upload images and videos

### PR DESCRIPTION
**Pull request by matt-morales**

### CODE REVIEW

**Comment by matt-morales** [comment]
@crcornwell helped me with this
We had to install two components: angular-uuid4 and angular-md5

**Review by davezuch — APPROVED** [review]
Pretty cool! 🚀

**Review by crcornwell — APPROVED** [review]

File: src/cn-media-upload.directive.js [hunk 1]
```diff
@@ -73,26 +73,48 @@
...
         url: vm.cnUploadPath,
-        headers: $http.defaults.headers.common,
+        headers,
         data: formData,
         processData: false,
         contentType: false,
         type: 'POST',
-        success: dfr.resolve,
+        success: (response) => {
+	  if (response.media_object) dfr.resolve(response)
```
Comment (davezuch) [inline src/cn-media-upload.directive.js:117]:
```suggestion
	   if (response.media_object) dfr.resolve(response)
```

File: src/cn-media-upload.directive.js [hunk 2]
```diff
@@ -73,26 +73,48 @@
...
+	  else if (start + step < size) uploadFile_(file, start + step, step, dfr, uuid, fileHash)
```
Comment (davezuch) [inline src/cn-media-upload.directive.js:120]:
```suggestion
	   else if (start + step < size) uploadFile_(file, start + step, step, dfr, uuid, fileHash)
```

**Review by davezuch — APPROVED** [review]

File: src/cn-media-upload.directive.js (hunk 1 above)
Comment (davezuch) [inline src/cn-media-upload.directive.js:117]:
```suggestion
	   if (response.media_object) dfr.resolve(response)
```

File: src/cn-media-upload.directive.js (hunk 2 above)
Comment (davezuch) [inline src/cn-media-upload.directive.js:120]:
```suggestion
	   else if (start + step < size) uploadFile_(file, start + step, step, dfr, uuid, fileHash)
```

**Review by davezuch — COMMENTED** [review]
Just trying to fix the spacing using Github's UI, it's confusing

**Pull request merged**